apps/client/src/app/shop/[shop_id]/_components/ShopProduct/ShopProduct.tsx

```diff
@@ -0,0 +1,47 @@
+import Link from "next/link";
+import { Flex, Text } from "@chakra-ui/react";
+
+import { Icon, ImageBox } from "~/components";
+
+import type { ProductInfo } from "../../_types";
+
+export const ShopProduct = (productInfo: ProductInfo) => {
+  const { id, name, topCategory, introduce, price } = productInfo;
+
+  return (
+    <Link href={`/shop/${id}/products`} style={{ position: "relative" }}>
+      <div style={{ position: "absolute", top: "8px", right: "8px" }}>
+        <Icon name="sell" />
+      </div>
```

Comment:
```suggestion
      <Box position="absolute" top="8px" right="8px">
        <Icon name="sell" />
      </Box>
```

차크라에는 Box라는 컴포넌트가 있습니다. 인라인 style은 컨벤션에 맞지 않게 휘뚜루마뚜루 느낌이 강해서 차크라의 Box를 사용해 보시는 건 어떨까요?